Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/ActionsLibraryNavPage.xaml.cs

```diff
@@ -59,10 +59,13 @@ public ActionsLibraryNavPage(Context context)
             InitializeComponent();
             mContext = context;
             mActionsList = mContext.BusinessFlow.CurrentActivity.Acts;
+
+            mContext.PropertyChanged += MContext_PropertyChanged;
+            mContext.Activity.PropertyChanged += Activity_PropertyChanged;
```

Comment:
mContext.Activity.PropertyChanged is needed or you can relay only on Context now?

Reply:
Actually, after merging with the latest change, i tried with mContext.Target over APIs and POMs pages but that wasn't working so i did not made further changes over the same branch. I'll update the same and push.